bugbug/models/tracking.py

```diff
@@ -100,13 +100,27 @@ def get_labels(self):
         for bug_data in bugzilla.get_bugs():
             bug_id = int(bug_data["id"])
 
-            for entry in bug_data["history"]:
-                for change in entry["changes"]:
-                    if change["field_name"].startswith("cf_tracking_firefox"):
-                        if change["added"] in ["blocking", "+"]:
-                            classes[bug_id] = 1
-                        elif change["added"] == "-":
-                            classes[bug_id] = 0
+            flag = False
+            keys = bug_data.keys()
+            tracking_list = [
```

Comment:
```suggestion
            tracking_flags = [
```

Reply:
Shall I create a new PR or commit chages in this PR directly?